public/doc/openapi-3-v1.0.0.json

```diff
@@ -2239,7 +2239,7 @@
           "403": {
             "$ref": "#/components/responses/Forbidden"
           },
-          "422": {
+          "400": {
```

Comment:
@lindgrenj6 Should we change the #/components/response/InvalidEntity to be BadRequest